__tests__/render.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../components/App.jsx';
+
+
+describe('First React component test with Enzyme', () => {
+   it('renders without crashing', () => {
+
+    shallow(<App />)    );
+});
```

Comment:
In the line 9 has an extra parenthesis after App component.
This test has no expectation `expect <something> to be <something>`

Reply:
The enzyme test stuff was kinda left on the back burner. I was working on my test then  I was told to work on my module instead. so instead of pushing this stuff and waiting for a pull request on broken code I kept working.